TestFoundation/TestNSJSONSerialization.swift

```diff
@@ -1054,6 +1058,26 @@ extension TestNSJSONSerialization {
         XCTAssertEqual(try trySerialize(json), "[false,true]")
     }
     
+    func test_serialize_IntMax() {
+        let json: [Any] = [Int.max]
```

Comment:
Thanks for finding that, I've updated the tests to compare against a string representation of the `Int.max` value so this should also work on 32-bit platforms now.